src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.tsx

```diff
@@ -99,7 +102,9 @@ const ConditionsList: React.FC<IConditionsListProps> = (props) => {
                         selectedCondition: valueRanges.find(
                             (vr) => vr.id === conditionId
                         ),
-                        selectedTarget: `#${LIST_KEY}`
+                        selectedTarget: `#${
+                            CONDITIONS_ITEM_ID_PREFIX + conditionId
```

Comment:
maybe just have a util that takes in the condition id and gives back a string? then you don't need the constant at module scope cause all the places can just call that util. And it means the generation logic is all in one spot.

`getListItemId`